recipes/esgf-pyclient/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "esgf-pyclient" %}
+{% set version = "0.1.6" %}
+{% set hash_type = "md5" %}
+{% set hash_val = "228ac9d7e3a600f587c7cb46c7389cdc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - jinja2
+
+test:
+  imports:
+    - pyesgf
+    - pyesgf.search
+    - pyesgf.security
+
+about:
+  home: http://esgf-pyclient.readthedocs.org
+  license: BSD 3-clause
```

Comment:
So maybe make a comment like this for now in the recipe should be sufficient. Feel free to change the wording as you see fit.

```yaml
  # Please remember to add the license file after the PR is merged and included in a release.
  # https://github.com/ESGF/esgf-pyclient/pull/7
  # license_file: LICENSE
```